test/NuGet.Core.Tests/NuGet.Configuration.Test/SettingsFileParsingTests/SourceItemTests.cs

```diff
@@ -54,14 +54,20 @@ public void SourceItem_ParsedSuccessfully()
 <configuration>
     <packageSources>
         <add key='nugetorg' value='http://serviceIndexorg.test/api/index.json' />
-        <add key='nuget2' value='http://serviceIndex.test/api/index.json' protocolVersion='3' />
+        <add key='nuget2' value='http://serviceIndex.test2/api/index.json' protocolVersion='3' />
+        <add key='nuget3' value='http://serviceIndex.test3/api/index.json' protocolVersion='3' allowInsecureConnections='true' />
+        <add key='nuget4' value='http://serviceIndex.test4/api/index.json' allowInsecureConnections='true' />
+        <add key='nuget5' value='http://serviceIndex.test5/api/index.json' allowInsecureConnections='false' protocolVersion='2' />
     </packageSources>
 </configuration>";
 
             var expectedValues = new List<SourceItem>()
             {
                 new SourceItem("nugetorg","http://serviceIndexorg.test/api/index.json"),
-                new SourceItem("nuget2","http://serviceIndex.test/api/index.json", "3" ),
+                new SourceItem("nuget2","http://serviceIndex.test2/api/index.json", "3" ),
+                new SourceItem("nuget3","http://serviceIndex.test3/api/index.json", protocolVersion: "3", allowInsecureConnections: "true" ),
```

Comment:
what happens if we specify `allowInsecureConnections` in a different case?

Reply:
Thanks for reviewing! Yes, it works for a different case. I changed one line to use Uppercase.